### DISCUSSION TITLE
Install instructions bit confused.

### DISCUSSION THREAD

**Discussion by afro-coder** [rose-pine/neovim#62] [Q&A] [answered]
Hey really new to nvim was migrating from vim, and the install instruction has this
```
use({
    'rose-pine/neovim',
    as = 'rose-pine',
    tag = 'v0.1.0', -- Optional tag release
    config = function()
        vim.cmd('colorscheme rose-pine')
    end
})
```
but I'm not able to figure out where this goes, I tried adding it to the init file and init.lua and it doesn't work can someone help I tried searching online too.

**Comment by mvllow** [comment, marked as answer]
Those instructions are for a lua plugin manager called packer.nvim. If you are using vim-plug, you can instead do `Plug 'rose-pine/neovim'` and after your plugins add `colorscheme rose-pine`

**Comment by afro-coder** [comment]
Ah gotcha thanks!

**Comment by mal2994** [comment]
I have lazy package manager for neovim... I tried but nothing worked until I created a new file like this`~/.config/nvim/lua/plugins/rose-pine.lua` and put this code in it:

```
return { "rose-pine/neovim", name = "rose-pine" }
```

Optionally, you can add this code to set the default colorscheme `~/.config/nvim/init.lua`

```
vim.cmd("colorscheme rose-pine")
```
